src/fides/api/ops/tasks/__init__.py

```diff
@@ -65,6 +65,17 @@ def _create_celery(config_path: str = CONFIG.execution.celery_config_path) -> Ce
 celery_app = _create_celery()
 
 
+def get_workers_health() -> Dict[str, Any]:
```

Comment:
doc string here that describes the return object format? I can't really parse easily what `health` ends up looking like